Libraries/PeriphDrivers/Source/AFE/hart_uart.c

```diff
@@ -503,6 +503,55 @@ static int hart_uart_pins_idle_mode(void)
     return retval;
 }
 
+static int hart_uart_pins_external_test_mode_state(void)
+{
+    int retval = 0;
+    mxc_gpio_cfg_t hart_pin;
+
+    // RTS Input to AFE, LOW is transmit mode, so Pulling Up
+    hart_pin.port = HART_RTS_GPIO_PORT;
+    hart_pin.mask = HART_RTS_GPIO_PIN;
+    hart_pin.pad = MXC_GPIO_PAD_PULL_UP;
+    hart_pin.func = MXC_GPIO_FUNC_IN;
+
+    retval = MXC_AFE_GPIO_Config(&hart_pin);
+    if (retval != E_NO_ERROR) {
+        return retval;
+    }
+
+    // CD output from AFE, Tristate
+    hart_pin.port = HART_CD_GPIO_PORT;
+    hart_pin.mask = HART_CD_GPIO_PIN;
+    hart_pin.pad = MXC_GPIO_PAD_NONE;
+
```

Comment:
hart_pin.func isn't set here. Seems wrong.

Reply:
I will update these to explicitly set the func register as well for clarity.